stripped/colabs/Anotate_stripped.py

```diff
@@ -0,0 +1,43 @@
+import random
+import pandas as pd
+import numpy as np
+
+from google.colab import drive
+drive.mount("/content/drive")
+
+NASA = pd.read_csv('/content/drive/MyDrive/sampled_NASA_200k.csv') #FIXME to powinien byc inny dataset np. single_month albo full
```

Comment:
Jeśli "full" to anotujemy współrzędne całkowicie w oderwaniu od czasu, tak samo dla wszystkich obserwacji z ponad 40 lat? Pewnie znalezienie takich danych dla różnych, bardziej odległych lat (np. sprzed 30-40 lat) jest praktycznie niemożliwe 😕 i tu jest pies pogrzebany 😕 i dlatego z góry zakładamy, że przez ponad 40 lat nic się nie zmieniło (oczywiście teoretycznie mogło i tak być) ?